src/plugins/saved_objects/public/save_modal/saved_object_save_modal.tsx

```diff
@@ -111,6 +112,11 @@ export class SavedObjectSaveModal extends React.Component<Props, SaveModalState>
               defaultMessage="Save {objectType}"
               values={{ objectType: this.props.objectType }}
             />
+            {this.props.objectType === 'Visualization (VisBuilder)' ? (
+              <EuiBadge color="primary">{'Experimental'}</EuiBadge>
+            ) : (
+              <></>
+            )}
```

Comment:
Why do we need to explicitly set an empty child node? We should be able to just conditionally render the badge without the ternary.

Reply:
will make it as a function that conditionally render the badge by checking the experimental flag